src/design-tokens/mixins.css

```
@@ -116,29 +116,21 @@
   /**
    * Input error state
    */
-  &.eds-is-error {
+  &.error {
     border-color: var(--eds-theme-color-border-utility-error-strong);
   }
 
   /**
    * Disabled state
    */
-  &.eds-is-disabled {
+  &:disabled {
     background-color: var(--eds-theme-color-background-disabled);
     border-color: var(--eds-theme-color-border-disabled);
     color: var(--eds-theme-color-text-disabled);
     cursor: not-allowed;
   }
 
-  &::-webkit-input-placeholder {
-    @mixin placeholderStyles;
-  }
-
-  &::-moz-placeholder {
-    @mixin placeholderStyles;
-  }
-
-  &:-ms-input-placeholder {
+  &::placeholder {
```

Could we use a single colon, for consistency?
```suggestion
  &:placeholder {
```

---

Think it needs 2 colons [mdn](https://developer.mozilla.org/en-US/docs/Web/CSS/::placeholder)

---

Awesome thanks!